server/src/controllers/index.ts

```diff
@@ -0,0 +1,4 @@
+import { login } from './user';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
avoid commented lines, instead you can add this rule to your rules in eslint file.